view/rest/handlers/comment.go

```diff
@@ -63,7 +64,8 @@ func (h *Handlers) GetComment(w http.ResponseWriter, r *http.Request) {
 
 	comment, err := h.service.GetComment(r.Context(), commentID)
 	if err != nil {
-		http.Error(w, "cannot get comment with such ID", http.StatusNotFound)
+		code := handleError(err)
+		http.Error(w, "cannot get comment", code)
```

Comment:
why don't put `http.Error(w, "cannot get comment", code)` in `handleError` to not copy everywhere?